src/StackedTableBody.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import css from 'styled-jsx/css'
+import propTypes from 'prop-types'
+import { instanceOfComponent } from '@dhis2/prop-types'
+
+import { StackedTableRow } from './StackedTableRow.js'
+
+const tableBodyStylesResponsive = css`
+    tbody {
+        display: block;
```

Comment:
I notice in the story that the text content inside the table is rendering as browser default. It should be `grey900`. Is this just the case because it is an isolated component? I'm thinking that `grey900` should be the `table` color defnition. (Perhaps that is done in the non-stacked table and then inherited here?)